Slim/Control/Commands.pm

```diff
@@ -380,8 +380,9 @@ sub clientForgetCommand {
 	# Persist playback state like we would do when turning off a player, that is, treat a vanishing
 	# player that's still playing the same way as a player that's turned off while still playing, so
 	# we can make it start playing again if it reappears and the user told us to resume playing
-	# when powering on.
-	$client->persistPlaybackStateForPowerOff();
+	# when powering on. Make sure elapsed is not 0 if we are playing
+	my $playing = $client->controller->playingSongElapsed() + 0.1 if $client->controller->isPlaying(1);
+	$prefs->client($client)->set('playingAtPowerOff', $playing || 0) if $client->power();
```

Comment:
Storing fractional values in a value whose name implies it's a boolean value seems slightly off, doesn't it?

Reply:
Yea and no Perl is not typed language so it really does not matter. What as I'm doing here is changing the meaning of that setting from what was indeed a bool to a float that says for how long have we been playing - 0 is we where not and anything else is the elapsed time. So it's not misusing a book, it's more like changing the signification. I can add another settings, but I usually prefer to have minimalistic representation of states rather that correlated variables.

Reply:
I understood all that, what I'm heading at is that anybody reading this code in a few years should still be able to understand it, and I'm not sure about that if a variable named 'playing' read from a pref named 'playingAtPowerOff' contains a playback position (in seconds? In samples?) ... it's just kinda obscure.

Reply:
I agree, I can try to add some better comment, but otherwise I'd have to add another variable. I can do that if you both prefer